### PR TITLE
Fix/aut 2/add multiple mime types

### PR DESCRIPTION
**Pull request by bartlomiejmarszal**
Adding multiple mime types saved in `type` attribute now. 

https://oat-sa.atlassian.net/browse/AUT-2

### CODE REVIEW

**Review by mike-ionut-mihai-sandu-tao — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

File: src/qtiCommonRenderer/helpers/uploadMime.js [hunk 1]
```diff
@@ -45,54 +47,45 @@ var uploadMime = {
...
-                    function(acc, selectedType) {
+                    function (acc, selectedType) {
                         return acc + ' x-tao-upload-type-' + selectedType.replace('/', '_');
                     },
                     classes
                 ).trim()
             );
         }
+
+        interaction.attr('type', types.join(delimetr));
```
Comment (bziondik) [inline src/qtiCommonRenderer/helpers/uploadMime.js:65]: For case `types.length === 1` no call `interaction.attr('type', types[0]);`

**Review by bziondik — CHANGES_REQUESTED** [review]
Please, check one case

File: src/qtiCommonRenderer/helpers/uploadMime.js (hunk 1 above)
Comment (AnastasiaArcadia) [reply src/qtiCommonRenderer/helpers/uploadMime.js:65]: For any length `interaction.attr('type', types.join(delimetr));` will be called which equals to `interaction.attr('type', types[0]);`

**Review by bziondik — COMMENTED** [review]
- [ ]  New code is covered by tests (if applicable)
- [ ]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [x]  New code is not subject to concurrency issues (if applicable)
- [ ]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

**Review by bziondik — CHANGES_REQUESTED** [review]
@AnastasiaArcadia please, check FE unit tests
image
I have PR with partially fixed unit tests, you can use it https://github.com/oat-sa/tao-item-runner-qti-fe/pull/148

**Review by bziondik — APPROVED** [review]
- [ ]  New code is covered by tests (if applicable)
- [x]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [x]  New code is not subject to concurrency issues (if applicable)
- [ ]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

**Pull request merged**